README.md

```diff
@@ -36,80 +36,172 @@ spec:
       }
     ]
   }'
+```
+
+The `example-network` specifies two CNI plugins, the first cni in the above is a `cnv-bridge`, connecting pods to linux-bridge `br1`, and the second
+is the `cidr-filtering-cni`, that handles configuration of nftables rules on the pod.
+
+By default, all ingress and egress traffic is dropped.
+To allow specific ingress and egress CIDR blocks/ports, create configMaps(s) referencing a NetworkAttachmentDefinition spec.config name in the configMaps label.
+Each configMap additionally needs to have `cidr-filtering-cni` label to be enabled for the cidr-filtering-cni plugin.
+
+The following manifest is an example of a minimal configMap, that doesn't yet allow any traffic. It's given as an example, since denying all traffic is done by default, it's
```

Comment:
```suggestion
The following manifest is an example of a minimal `configMap`, that doesn't yet allow any traffic. It's given as an example, since denying all traffic is done by default, it's
```